parche apra test lsb-release
Index: lsb/test/test_lsb_release.py
===================================================================
--- lsb.orig/test/test_lsb_release.py	2013-07-25 16:57:15.888461294 -0430
+++ lsb/test/test_lsb_release.py	2013-07-29 17:24:10.625265509 -0430
@@ -19,7 +19,7 @@
 
 def get_arch_distinfo():
 	# Copied verbatim from guess_debian_release; sucks but unavoidable.
-	distinfo = {'ID' : 'Debian'}
+	distinfo = {'ID' : 'Canaima'}
 	kern = os.uname()[0]
 	if kern in ('Linux', 'Hurd', 'NetBSD'):
 		distinfo['OS'] = 'GNU/'+kern
@@ -225,7 +225,7 @@
 
 			distinfo['CODENAME'] = lr.RELEASE_CODENAME_LOOKUP[rno]
 			distinfo['DESCRIPTION'] = '%(ID)s %(OS)s %(RELEASE)s (%(CODENAME)s)' % distinfo
-			fn = 'test/debian_version_' + rnd_string(5,5)
+			fn = 'test/canaima_version_' + rnd_string(5,5)
 			f = open(fn,'w')
 			f.write(distinfo['RELEASE'])
 			f.close()
@@ -240,7 +240,7 @@
 		for rno in lr.RELEASE_CODENAME_LOOKUP:
 			distinfo['RELEASE']  = lr.RELEASE_CODENAME_LOOKUP[rno]
 			distinfo['DESCRIPTION'] = '%(ID)s %(OS)s %(RELEASE)s' % distinfo
-			fn = 'test/debian_version_' + rnd_string(5,12)
+			fn = 'test/canaima_version_' + rnd_string(5,12)
 			f = open(fn,'w')
 			f.write(distinfo['RELEASE'])
 			f.close()
@@ -253,7 +253,7 @@
 		distinfo['RELEASE'] = 'testing/unstable'
 		distinfo['DESCRIPTION'] = '%(ID)s %(OS)s %(RELEASE)s' % distinfo
 		for rno in lr.RELEASE_CODENAME_LOOKUP:
-			fn = 'test/debian_version_' + rnd_string(5,12)
+			fn = 'test/canaima_version_' + rnd_string(5,12)
 			f = open(fn,'w')
 			f.write(lr.RELEASE_CODENAME_LOOKUP[rno] + '/sid')
 			f.close()
@@ -268,7 +268,7 @@
 		distinfo['RELEASE'] = 'unstable'
 		distinfo['DESCRIPTION'] = '%(ID)s %(OS)s %(RELEASE)s (%(CODENAME)s)' % distinfo
 		for rno in lr.RELEASE_CODENAME_LOOKUP:
-			fn = 'test/debian_version_' + rnd_string(5,12)
+			fn = 'test/canaima_version_' + rnd_string(5,12)
 			f = open(fn,'w')
 			f.write(lr.RELEASE_CODENAME_LOOKUP[rno] + '/sid')
 			f.close()
@@ -283,7 +283,7 @@
 		distinfo['RELEASE'] = 'unstable'
 		distinfo['DESCRIPTION'] = '%(ID)s %(OS)s %(RELEASE)s (%(CODENAME)s)' % distinfo
 		for rno in lr.RELEASE_CODENAME_LOOKUP:
-			fn = 'test/debian_version_' + rnd_string(5,12)
+			fn = 'test/canaima_version_' + rnd_string(5,12)
 			f = open(fn,'w')
 			f.write(lr.RELEASE_CODENAME_LOOKUP[rno] + '/sid')
 			f.close()
@@ -315,7 +315,7 @@
 		supposed_output['DESCRIPTION'] = '%(ID)s %(OS)s %(RELEASE)s' % supposed_output
 
 		os.environ['LSB_ETC_LSB_RELEASE'] = 'test/inexistant_file_' + rnd_string(2,5)
-		fn = 'test/debian_version_' + rnd_string(5,12)
+		fn = 'test/canaima_version_' + rnd_string(5,12)
 		f = open(fn,'w')
 		f.write('testing/sid')
 		f.close()
